CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [1.12.11]
+### Fixed
+- Offline plotting now works in other domains besides `plot.ly`. In addition, the `link text` in the bottom right corner of the offline plots properly display `Export to [Domain Name]` for the given Domain name of the server.
```

Comment:
ahh this may seem nitpicky but offline plotting doesn't work 'in' a given domain (cause it's offline), it was just an issue with the link text not getting assigned at the right time. This could just be:
The `link text` in the bottom right corner of offline plots now properly displays `Export to [Domain Name]` for the given domain set in the user's .config file.